apps/etterlatte-saksbehandling-ui/client/src/components/behandling/brev/Vedtaksbrev.tsx

```diff
@@ -28,9 +28,7 @@ export const Vedtaksbrev = (props: { behandling: IDetaljertBehandling }) => {
     if (!behandlingId || !sak) return
 
     const fetchVedtaksbrev = async () => {
-      const skalIkkeHenteVedtaksbrev =
-        props.behandling.behandlingType === IBehandlingsType.REVURDERING &&
-        props.behandling.prosesstype === IProsesstype.AUTOMATISK
+      const skalIkkeHenteVedtaksbrev = props.behandling.behandlingType === IBehandlingsType.REVURDERING
```

Comment:
Lurer på om dette blir feil, for tanken med den opprinnelege var primært å filtrere bort (automatisk) regulering, for mange andre revurderingstypar skal vi jo hente vedtaksbrev

Reply:
Vet ikke om du skrev dette før eller etter jeg stilte spørsmål på Slack. 

`prosesstype` er _alltid_ undefined; ergo vil denne funksjonen alltid være `false`. Så den har aldri fungert.

Reply:
Ergo var denne før alltid `false` for revurdering, no er den alltid `true` for revurdering

Reply:
Iom. at det tydeligvis har fungert til nå med `false` kan det hende det er best å bare fjerne det helt. 

Jeg tror funksjonen `revurderingRoutes` i `BehandlingRoutes.tsx` er det som "løser problemet", eller spesifikt `behandlingSkalSendeBrev()` under `utils.ts`: 

```
export const behandlingSkalSendeBrev = (behandling: IBehandlingReducer): boolean => {
  switch (behandling.behandlingType) {
    case IBehandlingsType.FØRSTEGANGSBEHANDLING:
      return true
    case IBehandlingsType.MANUELT_OPPHOER:
      return false
    case IBehandlingsType.REVURDERING:
      return !(
        behandling.revurderingsaarsak === Revurderingsaarsak.REGULERING ||
        behandling.revurderingsaarsak === Revurderingsaarsak.DOEDSFALL
      )
  }
}
```

Reply:
Ja, støttar å ta det bort.

Reply:
Ved nærmere ettertanke tok jeg heller i bruk `behandlingSkalSendeBrev()`. Så har vi en liten backup på brevsiden i tilfelle man skulle klare å navigere dit ved uhell.